src/runtime_src/core/tools/common/tests/TestIPU.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+#include "TestIPU.h"
+#include "tools/common/XBUtilities.h"
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_hw_context.h"
+#include "xrt/xrt_kernel.h"
+namespace XBU = XBUtilities;
+
+#include <boost/filesystem.hpp>
+
+static constexpr size_t host_app = 1; //opcode
+static constexpr size_t buffer_size = 128;
+
+// ----- C L A S S   M E T H O D S -------------------------------------------
+TestIPU::TestIPU()
+  : TestRunner("ipu-verify", 
```

Comment:
Is this ipu verify or AIE verify? Is the 1x4 xclbin usable on any other platforms?

Reply:
discussed offline

Reply:
Actually I am still concerned about the name of this test. Have you talked to PRT about the name?